### PR TITLE
feat: add eslint plugin

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Description
This module adds eslint as static files and updates package.json.

### Fixes
closes amplication/amplication#6605

### CODE REVIEW

File: plugins/linter-eslint/README.md
Comment (morhag90) [inline plugins/linter-eslint/README.md:1]: @Shurtu-gal  Please update this file with the relevant information

File: plugins/linter-eslint/src/events/createAdminUI.ts
Comment (morhag90) [inline plugins/linter-eslint/src/events/createAdminUI.ts:1]: @Shurtu-gal  Looks like the same logic as in the afterCreate server function. Please write one function for both events (and pass the baseDirectory according to the event: server/client).

**Review by morhag90 — CHANGES_REQUESTED** [review]

File: plugins/linter-eslint/README.md
Comment (Shurtu-gal) [reply plugins/linter-eslint/README.md:1]: Sure

File: plugins/linter-eslint/src/events/createAdminUI.ts
Comment (Shurtu-gal) [reply plugins/linter-eslint/src/events/createAdminUI.ts:1]: Okay

**Review by morhag90 — APPROVED** [review]

**Comment by morhag90** [comment]
@Shurtu-gal  Great work!!

**Comment by Shurtu-gal** [comment]
Thanks

**Pull request merged**